libevmasm/AssemblyItem.h

```diff
@@ -162,6 +176,9 @@ class AssemblyItem
 	AssemblyItemType m_type;
 	Instruction m_instruction; ///< Only valid if m_type == Operation
 	std::shared_ptr<u256> m_data; ///< Only valid if m_type != Operation
+	/// If m_type == VerbatimBytecode, this holds number of arguments, number of
+	/// return variables and verbatim bytecode.
+	std::optional<std::tuple<size_t, size_t, bytes>> m_verbatimBytecode;
```

Comment:
Oh, this got increased to three members? I think it should really be documented what they are, I assume `args`,`rets`,`data`?

Reply:
Two lines up?